requirements/requirements-optionals.txt

```diff
@@ -2,3 +2,4 @@
 markdown==2.5.2
 django-guardian==1.3.2
 django-filter==0.10.0
```

Comment:
It is not directly imported, but for timezones to work properly it is required. The primary case I know of where this code would be used would be via [django-timezone-field](https://github.com/mfogel/django-timezone-field), which does require pytz. Also, Django [strongly recommends](https://docs.djangoproject.com/en/1.9/topics/i18n/timezones/#faq) installing pytz for timezone support.